app/controllers/event_notes_controller.rb

```diff
@@ -0,0 +1,82 @@
+class EventNotesController < ApplicationController
+  include SerializeDataHelper
+
+  rescue_from Exceptions::EducatorNotAuthorized, with: :redirect_unauthorized!
+
+  before_action :authorize!, except: [:names]
+
+  def authorize!
+    student = Student.find(params[:student_id])
+    educator = current_educator
+    raise Exceptions::EducatorNotAuthorized unless educator.is_authorized_for_student(student)
+  end
+
+  def create
+    clean_params = params.require(:event_note).permit(*[
```

Comment:
Factored out the params so I could share them between create and update. Thanks for the tip!